cmd/kube-etcd-signer-server/serve.go

```diff
@@ -62,8 +62,10 @@ func validateServeOpts(cmd *cobra.Command, args []string) error {
 		return errors.New("no signer CA flags passed one cert/key pair is required")
 	}
 
-	if serveOpts.sCrtFile == "" || serveOpts.sKeyFile == "" {
-		return errors.New("both --servcrt and --servkey are required flags")
+	if cl, kl := len(serveOpts.sCrtFiles), len(serveOpts.sKeyFiles); cl == 0 || kl == 0 {
+		return errors.New("at least one pair of --servcrt and --servkey is required")
+	} else if cl != kl {
+		return errors.New("mismatch pairs of --servcrt and --servkey")
```

Comment:
This might possibly be read as "key A is not a valid key for cert A".  Maybe rephrase to:

```go
return errors.Errorf("%d --servercrt does not match %d --servkey", cl, kl)
```

or something?